README.md

```diff
@@ -171,24 +176,24 @@ assert %{status: 403, message: "not authorized"} = error
 
 ## Schema Scopes
 
-Bodyguard also provides the `Bodyguard.Schema` behaviour to query which items a user can access. Implement it directly on schema modules.
+Bodyguard also provides the `Bodyguard.Schema` behaviour to filter out items a user shouldn't access. Implement it directly on schema modules.
 
 ```elixir
 defmodule MyApp.Blog.Post do
   import Ecto.Query, only: [from: 2]
   @behaviour Bodyguard.Schema
```

Comment:
I was actually intentionally avoiding this since a change introduced into the `__using__` might break a schema and the user might not know why - was just trying to touch the schemas as little as possible and force the user to change it from `@behaviour` to `use` if additional functionality gets added that warrants it, so they at least KNOW where the change comes from.